### DISCUSSION TITLE
How to use extract_sup_data.py

### DISCUSSION THREAD

**Discussion by kin0303** [NVIDIA-NeMo/NeMo#5006] [Q&A] [answered]
How to use scripts/dataset_processing/tts/extract_sup_data.py?

**Comment by godspirit00** [comment, marked as answer]
Prepare a manifest JSON file, then `cd` into your NeMo repo dir, and run `python scripts/dataset_processing/tts/extract_sup_data.py manifest_filepath=/path/to/your/manifest.json`, where `/path/to/your/manifest.json` should be replaced with the path to your manifest JSON file.
